web/cobrands/fixmystreet/assets.js

```diff
@@ -258,29 +258,52 @@ fixmystreet.add_assets = function(options) {
         }
 
         // An interactive layer for selecting a street light
-        var protocol_options = {
-            version: "1.1.0",
-            url: options.wfs_url,
-            featureType: options.wfs_feature,
-            geometryName: options.geometryName
-        };
-        if (fixmystreet.wmts_config) {
-            protocol_options.srsName = fixmystreet.wmts_config.map_projection;
-        }
-        if (options.propertyNames) {
-            protocol_options.propertyNames = options.propertyNames;
+        var protocol_options;
+        var protocol;
+        if (options.http_options !== undefined) {
+            protocol_options = OpenLayers.Util.extend(options.http_options, {});
+            if (protocol_options.format_class) {
+                protocol_options.format = new protocol_options.format_class(protocol_options.format_options);
+            } else {
+                protocol_options.format = new OpenLayers.Format.GML({
+                    featureNS: "http://mapserver.gis.umn.edu/mapserver",
+                    geometryName: options.geometryName
+                });
+            }
+            protocol = new OpenLayers.Protocol.HTTP(protocol_options);
+            protocol.srsInBBOX = true;
```

Comment:
Is there a reason this is set here rather than in the options? Not sue I follow the code flow entirely, but I think this means it's actually false for the purposes of the BBOX in the URL requested (as it's only read during object setup).

Reply:
You're right, this didn't actually do anything. I'd left it in accidentally.